rules.bzl

```diff
@@ -42,12 +42,15 @@ def _android_ndk_repository_impl(ctx):
 
     api_level = ctx.attr.api_level or 31
 
-    lib64_clang_directory = clang_directory + "/lib64/clang"
-    lib64_clang_files = ctx.path(lib64_clang_directory).readdir()
+    lib64_clang_directory = ctx.path(clang_directory + "/lib/clang")
+    if not lib64_clang_directory.exists:
+        fail("Expected directory %s to exist" % lib64_clang_directory)
+    lib64_clang_files = lib64_clang_directory.readdir()
+    lib64_clang_files = [lib64_clang_files[0]]
```

Comment:
note this is wrong, but right now it has `17` and `17.0.2`, which are _not_ symlinks of each other

Reply:
I fixed this by sorting the directory and picking a consistent one